packages/aws-cdk/test/api/hotswap/hotswap-test-setup.ts

```diff
@@ -107,6 +109,13 @@ export class HotswapMockSdkProvider {
     this.mockSdkProvider.stubGetEndpointSuffix(stub);
   }
 
+  public registerCloudWatchLogGroups(
+    stackArtifact: cxapi.CloudFormationStackArtifact,
+    cloudWatchLogMonitor: CloudWatchLogEventMonitor,
+  ): Promise<void> {
+    return registerCloudWatchLogGroups(this.mockSdkProvider, stackArtifact, cloudWatchLogMonitor);
+  }
+
```

Comment:
Hmmm. Any reason this method is on this helper?

This is used for hotswap tests, and the log monitoring isn't really tied to hotswapping anymore.